consoles/sshVirtsh.pm

```diff
@@ -676,7 +676,7 @@ sub run_cmd ($self, $cmd, %args) {
 }
 
 sub run_cmd_retrying_on_timeouts ($self, $command, @args) {
-    my $attempts = $bmwqemu::vars{SVIRT_ASSET_DOWNLOAD_ATTEMPTS} // 3;
+    my $attempts = $bmwqemu::vars{SVIRT_ASSET_DOWNLOAD_ATTEMPTS} // 1;
```

Comment:
but this disables retrying which we put in on purpose. What will be the effect of this then?

Reply:
It means we're back to the problem where jobs fail after running into the already generous timeout. It didn't happen that often and I haven't seen these kinds of failures in the last 2 days (but only checked a small number of jobs).

Reply:
But shouldn't we have retries to cover short service outages?

Reply:
I guess retries *with backoff* would make sense to cover service outages. The retries implemented here have no backoff so if there's an outage for e.g. 10 seconds the retry here would not be of much use and just provoke the connection error 3 times. Well, actually it wouldn't because the retry done here is only done on timeouts anyway.

I really only introduced the retry very recently to improve the case when this runs into a timeout as this was the problem reported in the ticket I was working on. (Hence also the function name `…_on_timeouts`.)

Reply:
ok. I am still not quite sure that this is the right approach to have retries supported in code but not used by default anywhere but you are the expert :)

Reply:
I don't want to remove the retry completely because I might be able to make it actually work at some point, see https://github.com/os-autoinst/os-autoinst/pull/2682. That PR needs further experimentation, though.